include/dlaf_c/desc.h

```diff
@@ -0,0 +1,26 @@
+//
+// Distributed Linear Algebra with Future (DLAF)
+//
+// Copyright (c) 2018-2023, ETH Zurich
+// All rights reserved.
+//
+// Please, refer to the LICENSE file in the root directory.
+// SPDX-License-Identifier: BSD-3-Clause
+//
+
+#pragma once
+
+/// @file
+
+/// DLA-Future matrix descriptor
+struct DLAF_descriptor {
+  int m;     ///< Number of rows in the global matrix
+  int n;     ///< Number of columns in the global matrix
+  int mb;    ///< Row blocking factor
+  int nb;    ///< Column blocking factor
+  int isrc;  ///< Process row of the first row of the global matrix
+  int jsrc;  ///< Process column of the first column of the global matrix
+  int i;     ///< First row of the submatrix within global matrix, has to be 1
+  int j;     ///< First column of the submatrix within global matrix, has to be 1
```

Comment:
Last comment: it is not clear to me why 1.

This is not SCALAPACK dependent but C-API, so I would say we might want it 0-based. Is there any reason why is it better to have them 1-based?